tests/test_models/test_engine/test_db_storage.py

```diff
@@ -99,9 +99,9 @@ def test_db_func_get(self):
     @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
     def test_db_func_count(self):
         """Test for the count function"""
-        n_state = models.storage.count("State")
+        n_state = models.storage.count('State')
```

Comment:
i see that you change form double quotations to single to respect miss rico, great consideration!